Source/missiles.cpp

```diff
@@ -3604,12 +3604,8 @@ void MI_Infra(Missile &missile)
 
 void MI_Apoca(Missile &missile)
 {
-	int id = missile._misource;
-	bool exit = false;
-	int j;
-	int k;
-	for (j = missile.var2; j < missile.var3 && !exit; j++) {
-		for (k = missile.var4; k < missile.var5 && !exit; k++) {

```

Comment:
Removing `&& !exist` has a side-effect.
Previous `j` or `k` was increment once more.
This means `var4` and `var6` don't change if the missile hits.
But this is needed to move the Apocalypse spell to the next target.
In this pr the monster will be hit multiple times.

Reply:
Yikes! Thanks for catching this.